src/domain/template/lifecycle-template/lifecycle.template.service.ts

```diff
@@ -83,4 +84,44 @@ export class LifecycleTemplateService {
   ): Promise<ILifecycleTemplate> {
     return await this.lifecycleTemplateRepository.save(lifecycleTemplate);
   }
+
+  public async isLifecycleTemplateInHub(
+    lifecycleTemplateID: string,
+    hubID: string,
+    templateType: string
+  ) {
+    const [queryResult]: {
+      hubId: string;
+    }[] = await getConnection().query(
+      `
+      SELECT \'hub\'.\'id\' as \'hubId\' FROM \'hub\'
```

Comment:
You can have the result of the query (e.g. SELECT COUNT + WHERE hubId =...) already showing you whether isLifecycleTemplateInHub.